askbot/conf/access_control.py

```diff
@@ -15,7 +15,7 @@
     livesettings.BooleanValue(
         ACCESS_CONTROL,
         'READ_ONLY_MODE_ENABLED',
-        default=False,
+        default=True,
```

Comment:
Do we need this change? This seems to be changing the default (ie. when it's not already set) so shipping it won't have any impact.

Could we just set that value through the Askrover admin if we think it's important to do? I guess there's probably value in removing this message, though wouldn't we have more places where we show a message like this? (for example when you attempt to add a new question or answer; or are those not concerning because we're removing those links anyways?).

Reply:
I was thinking of setting this value to true too but I think we need to ask Infra first, I did this because I thought it was a good idea to set the default to True

about the message, the other places where I found it is in sections we removed like where we comment or add an answer

Reply:
I don't have any concerns related to this, I said that we should ask infra because they are the ones that manage the secrets (that's what I understood last time I wanted to change one). About changing the default value I can undo it I don't have any opinions on doing it or not.

Reply:
Got it. I don't think we have to here since these are not secrets -- these are settings for Askrover project. Since we own it, we can change them as we see fit. We can leave the default change in, I don't think it's going to do any harm.